lib/client/chart.js

```diff
@@ -187,6 +206,7 @@ function init (client, d3, $) {
     // get the height of each chart based on its container size ratio
     var focusHeight = chart.focusHeight = chartHeight * .7;
     var contextHeight = chart.contextHeight = chartHeight * .2;
+    var basalsHeight = chart.basalsHeight = focusHeight / 4;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['basalsHeight' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1129941902/issues/source?bid=2452379&fileBranchId=2886508#l209)